pkg/tsdb/grafana-pyroscope-datasource/instance.go

```diff
@@ -97,14 +102,17 @@ func (d *PyroscopeDatasource) profileTypes(ctx context.Context, req *backend.Cal
 func (d *PyroscopeDatasource) labelNames(ctx context.Context, req *backend.CallResourceRequest, sender backend.CallResourceResponseSender) error {
 	res, err := d.client.LabelNames(ctx)
 	if err != nil {
+		logger.Error("Received error from client", "error", err)
```

Comment:
Would be great to also mention the method name in the error i.e. `LabelNames` in this case so we can separate it from other errors. Same in other places.